src/pages/home/Home.js

```diff
@@ -1,9 +1,16 @@
 import { Alert, Button } from "antd";
 import React from "react";
+import { Helmet } from "react-helmet";
 
 const Home = () => {
   return (
     <>
+      <Helmet>
+        <title>Home | Deuvox</title>
+        <meta name="title" content="Home | Deuvox" />
```

Comment:
I don't think we need meta title on our web. Please refer to: https://stackoverflow.com/questions/21076201/difference-between-meta-name-title-tag-and-title-title-tag